github/licenses.go

```diff
@@ -10,23 +10,49 @@ import "fmt"
 // LicensesService handles communication with the license related
 // methods of the GitHub API.
 //
-// GitHub API docs: http://developer.github.com/v3/pulls/
+// GitHub API docs: https://developer.github.com/v3/licenses/
 type LicensesService service
 
+// RepositoryLicense represents the license for a repository.
+type RepositoryLicense struct {
+	Name *string `json:"name,omitempty"`
+	Path *string `json:"path,omitempty"`
+
+	SHA         *string `json:"sha,omitempty"`
+	Size        *int    `json:"size,omitempty"`
+	URL         *string `json:"url,omitempty"`
+	HTMLURL     *string `json:"html_url,omitempty"`
+	GitURL      *string `json:"git_url,omitempty"`
+	DownloadURL *string `json:"download_url,omitempty"`
+	Type        *string `json:"type,omitempty"`
+	Content     *string `json:"content,omitempty"`
+	Encoding    *string `json:"encoding,omitempty"`
+	Links       *struct {
```

Comment:
Yes, the only other place I see it is in the `Feeds` struct and it has more pertinent information there -- fine with removing this. Will do so and update PR.